stdlib/public/core/StringLegacy.swift

```diff
@@ -63,12 +63,6 @@ extension String {
   }
 }
 
-extension String {
-  public init(_ _c: UnicodeScalar) {
```

Comment:
Seems like you're also implementing another SE proposal here... Please put this one back to keep things focused.